docs/usage.rst

```diff
@@ -53,6 +53,16 @@ documentation: :py:func:`tedana.workflows.t2smap_workflow`.
 
 .. _constructing ME-EPI pipelines:
 
+Run post_taskcorr
+-----------------
+This workflow has to be called after tedana has been run on your multi-echo dataset.
+It uses timeseries that are known to be noise or signal of interest, and it computes
+`Normalised Cross Correlation`_ between them and the IC signals, to automatically reject or accept
+components.
+It overwrites the original comp_table_ica.txt, adding specific rationale labels (I098 and I099).
+
+.. _Normalised Cross Correlation: https://en.wikipedia.org/wiki/Cross-correlation#Normalized_cross-correlation_(NCC)
```

Comment:
You should add the argparse auto-generated documentation here.

Reply:
That's where I get kind of lost (i.e. I still don't know ho to do it! But I'm studying it). I'm going to look into it tomorrow and hope I can come back on that.

Thank you!

Reply:
Ok I think I did it!